recipes/efa_install.rb

```diff
@@ -42,8 +42,8 @@
 end
 
 installer_options = "-y"
-# efa-kmod currently unavailable for ARM instances
-installer_options += " -k" if arm_instance?
+# efa-kmod currently unavailable on Centos8 ARM
```

Comment:
I'd remove the "Centos8 arm" comment from here, this information is in the conditions file.